base_layer/core/src/validation/error.rs

```diff
@@ -59,8 +59,8 @@ pub enum ValidationError {
     InvalidOutputMr,
     #[error("Invalid kernel merkle root")]
     InvalidKernelMr,
-    #[error("Invalid range proof merkle root")]
-    InvalidRangeProofMr,
+    #[error("Invalid witness merkle root")]
+    InvalidWitnessMr,
```

Comment:
This variant isnt used (assuming everything compiles) and can be removed. Did a quick search and same for `InvalidKernelMr` and `InvalidOutputMr` can be removed

Reply:
...and `WeakerAccumulatedDifficulty`, they are all handled in `BlockError(..)` and `PowError(...)`